string in

---
int32 wallSequence
int32 robotPositionId
int32 bitmapId
string mapType
string buildingEditorPath
string out